assets/components/radioToggle/radioToggle.jsx

```diff
@@ -31,9 +32,11 @@ export default function RadioToggle(props: PropTypes) {
   const radioButtons = props.radios.map((radio: Radio, idx: number) => {
 
     const radioId = `${props.name}-${idx}`;
+    const showAnnual = getQueryParameter('showAnnual', 'false');
+    const className = showAnnual === 'true' ? 'component-radio-toggle__button--with-annual' : 'component-radio-toggle__button--without-annual';
```

Comment:
[Modifiers in BEM](http://getbem.com/naming/) are applied as an extra class, in addition to the block/element class:

> Modifier is an extra class name which you add to a block/element DOM node. Add modifier classes only to blocks/elements they modify, and keep the original class

So this would be `component-radio-toggle__button component-radio-toggle__button--with-annual`. A little verbose, but it means that [here](https://github.com/guardian/support-frontend/pull/211/files#diff-c2cd7a3b0d969819dbc6a6cc71ae7f40R76) and [here](https://github.com/guardian/support-frontend/pull/211/files#diff-c2cd7a3b0d969819dbc6a6cc71ae7f40R84) you should be able to keep `.component-radio-toggle__button`, because the CSS there isn't really a modifier, it's applied to the base element.

We have a [`generateClassName`](https://github.com/guardian/support-frontend/blob/master/assets/helpers/utilities.js#L21) function for this 💪.